src/lib/config.js

```diff
@@ -55,26 +59,57 @@ function parseKeysConfig () {
         secret: 'iMx6i3D3acJPc4aJlK0iT/pkJP3T+Dqte9wg6hXpXEv08CpNQSm1J5AI6n/' +
           'QVBObeuQWdQVpgRQTAJzLLJJA/Q==',
         public: '9PAqTUEptSeQCOp/0FQTm3rkFnUFaYEUEwCcyyySQP0='
+      },
+      notification_sign: {
+        secret: fs.readFileSync(path.join(__dirname, '../../test/data/signKeyRSAPrv.pem'), 'utf8'),
+        public: fs.readFileSync(path.join(__dirname, '../../test/data/signKeyRSAPub.pem'), 'utf8')
+      }
+    }
+  } else {
+    const privateKeyPath = Config.getEnv(envPrefix, 'SIGNING_PRIVATE_KEY')
+    const publicKeyPath = Config.getEnv(envPrefix, 'SIGNING_PUBLIC_KEY')
+    if (!privateKeyPath || !publicKeyPath) {
+      if (process.env.NODE_ENV === 'production') {
+        throw new Error(envPrefix.toUpperCase() + '_SIGNING_PRIVATE_KEY and ' +
+          envPrefix.toUpperCase() + '_SIGNING_PUBLIC_KEY must be provided')
+      }
+      log.info('Using autogenerated keys for notification signing.')
```

Comment:
haha ... I originally had this at warn level, then changed it to info. I'm fine with either. I'll update to `warn`, then merge the PR.